package.json

```diff
@@ -86,6 +88,13 @@
     "what-input": "^5.2.10"
   },
   "devDependencies": {
+    "@babel/core": "~7.15.8",
+    "@storybook/addon-actions": "^6.3.12",
+    "@storybook/addon-essentials": "^6.3.12",
+    "@storybook/addon-links": "^6.3.12",
+    "@storybook/builder-webpack5": "^6.3.12",
+    "@storybook/manager-webpack5": "^6.3.12",
+    "@storybook/react": "^6.3.12",
```

Comment:
It's such a shame that there is no other format for `package.json` like `package.yaml` or so,
so we can get comments (json will never support comments). In order to fully appreciate your PR I need to look up all the different packages while a single comment directly at the package would explain so much more. 

The plugins look like the defaults one, nothing special besides the webpack 5 thing required for gatsby/webpack ?

Reply:
For adding `storybook` initially, I ran [`npx sb init`](https://storybook.js.org/docs/react/get-started/install) which took care of all the heavy lifting. It figured out that this was a React project and added most of these dependencies, added the scripts in the `packge.json` and created an initial set of configurations. Really nice DX 🙌